_posts/2021-08-22-release-2.0.0.md

```diff
@@ -0,0 +1,162 @@
+---
+layout: post
+title: Version 1
+---
+
+We are happy to release version 2.0.0 of MDAnalysis!
+
+This is a new major version, includes major API changes and concludes our current [roadmap][].
+
+This version supports Python 3.6 through to 3.9 on Windows (32- and 64-bit), MacOS, and Linux. It also offers preliminary support for ppc64le and ARM64, but does not currently support Apple M1 processors.
```

Comment:
Support for legacy versions of Python is only available in MDAnalysis 1.x.

Reply:
It also contains a number of breaking changes in addition to a large number of updates and enhancements.